orca-bakery/src/test/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTaskSpec.groovy

```diff
@@ -605,6 +611,26 @@ class CreateBakeTaskSpec extends Specification {
     true                       | BakeRequest.CloudProviderType.aws
   }
 
+  @Unroll
+  def "sets bakeExists flag to #bakeExists when status is #status"() {
+    given:
+    task.bakery = Stub(BakeryService) {
+      createBake(*_) >> Observable.from(completedStatus)
```

Comment:
This test seems off to me. I would have expected this to use `status` instead of `completedStatus`.